config-connector/solutions/iam/kpt/pubsub-subscription/README.md

```diff
@@ -0,0 +1,38 @@
+Pub/Sub Subscription
+==================================================
+# NAME
+  pubsub-subscription
+# SYNOPSIS
+  This package configures permissions for a pubsub subscription by creating an IAMPolicyMember resource.
```

Comment:
This should be the same with the the shortDescription in the kptfile.

Reply:
They should be the same now.

Reply:
Resolving. Could you actually resolve the comment when you believe the problem is fixed?